hunts/src/AddPuzzleModal.js

```diff
@@ -1,14 +1,17 @@
 import React from "react";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
 import { addPuzzle } from "./puzzlesSlice";
 import { hideModal } from "./modalSlice";
+import { selectHuntTags } from "./huntSlice";
 
 function AddPuzzleModal({ huntId }) {
+  const allTags = useSelector(selectHuntTags);
   const [name, setName] = React.useState("");
   const [url, setUrl] = React.useState("");
+  const [assignedMeta, setAssignedMeta] = React.useState("none");
```

Comment:
What's wrong with using a falsy value for this?

Reply:
Nothing. Should I use the falsy value for the none option too?